heppy/Daemon.py

```
@@ -159,6 +161,13 @@ def connect_internal(self):
         config['dir'] = self.config.get_dir()
         self.client = REPP(self.config['epp'])
 
+    def logout(self, args=None):
```

Ruff (0.14.8)

Unused method argument: `args`

(ARG002)